src/mrack/providers/beaker.py

```diff
@@ -277,16 +277,20 @@ def prov_result_to_host_data(self, prov_result, req):
     def _get_recipe_info(self, beaker_id, log_msg_start):
         """Get info about the recipe for beaker job id."""
         bkr_job_xml = self.hub.taskactions.to_xml(beaker_id).encode("utf8")
-
+        logs_dict = {}
         resources = []
         for recipe in eTree.fromstring(bkr_job_xml).iter("recipe"):
+            for logs in recipe.iter('logs'):
+                    for log in logs.iter('log'):
```

Comment:
There is an indentation error: https://dev.azure.com/neoave/mrack/_build/results?buildId=1135&view=logs&j=e4354d11-29aa-59c2-ce04-0d9aced3ea05&t=37a28340-749f-5255-d3d8-b83a7b4239ca&l=16